probe-rs/src/debug/mod.rs

```diff
@@ -943,8 +944,64 @@ impl<'debuginfo> UnitInfo<'debuginfo> {
                         ));
                     }
                 },
+                //Property of variables that are of DW_TAG_subrange_type
+                gimli::DW_AT_lower_bound => match attr.value() {
+                    gimli::AttributeValue::Data1(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
+                    gimli::AttributeValue::Data2(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
+                    gimli::AttributeValue::Data4(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
+                    gimli::AttributeValue::Data8(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
+                    gimli::AttributeValue::Sdata(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
+                    gimli::AttributeValue::Udata(value) => {
+                        child_variable.range_upper_bound = value as i64
+                    }
```

Comment:
Yeah, I saw that, but the docs weren't clear on how it would handle udata values. After your comment, I drilled into their source code, and they basically do the match I do here, so I will replace my code.